### DISCUSSION TITLE
After loading a scene from JSON, the images don't render.

### DISCUSSION THREAD

**Discussion by alandbh** [excalidraw/excalidraw#6255] [Q&A]
Hi everyone!

I'm using the `<Excalidraw />` component in my Nexj.JS application.

Everything is working just fine, but when I try to load a scene from [this JSON](https://excalidraw-custom-board.vercel.app/api/scene), only shapes, texts and lines are rendered properly - not images.

### This is the original scene: 
I exported this scene by clicking on "Save to..." button in main menu, and then, changed its extension to `.json`.

<img width="887" alt="image" src="https://user-images.githubusercontent.com/1545896/219423424-c3e27452-5ed3-4f15-8a8d-72f4643a71c1.png">

### Simutating the issue.

If you click on the blue button above ("Import JSON and update Scene"), you'll see something like the following:

<img width="888" alt="image" src="https://user-images.githubusercontent.com/1545896/219426171-4eadcd09-8adc-4c08-9433-d88a1eba86eb.png">

This is the function for loading the scene:

```
const updateScene = () => {
        if (excalidrawAPI !== null) {
            excalidrawAPI.updateScene(jsonData);
        }
    };

<button
    className="px-4 py-2 border-blue-500 border-solid border-2 rounded-md text-blue-500"
    onClick={updateScene}
>
```
**The funny part:** If I just drag and drop the same JSON file over the canvas, the images work just fine 🤔

**Dou you guys, have any idea of how to solve this? Am I missing something?**

Btw, thanks to Excalidraw team for this amazing library! 😊

**Comment by alandbh** [comment]
### You can use the application here:
[https://excalidraw-custom-board.vercel.app/board](https://excalidraw-custom-board.vercel.app/board)

**Reply by LukaHietala** [reply]
It doesn't exist

**Comment by oskarlf** [comment]
I think there is no way to add image such way like yours. The only way i found out how to do it is to pass json elements and files to `initialData` parameter.
